docs/vgclog.md

```diff
@@ -0,0 +1,207 @@
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Verbose Garbage Collection Logs 
+
+[Garbage collection](https://www.eclipse.org/openj9/docs/gc/)(gc) reclaims unused memory in the heap for reuse. During cleanup of the heap, the verbose garbage collection (vgc) logs capture information about the different garbage collection operations that are involved in the garbage collection cycles. Gc operations include GC initialization, stop-the-world processing, finalization, reference processing, and allocation failures. 
+
+A garbage collection event is triggered in the following situations: 
+
+- The JVM calls system.gc() to run a global STW gc. 
+
+- A memory allocation failure. Allocation failures are triggered when a request to allocate an object fails due to a full region of the heap. 
+
+- Decrease in available memory to a value that triggers a concurrent cycle to clean the heap or a region of the heap. 
+
+- Concurrent events that are part of a concurrent cycle are complete, which triggers an STW event that completes either an incremental or final part of the concurrent.
+
+Verbose garbage collection (vgc) logs contain information on garbage collection operations to assist with the following actions:  
+
+- Tuning the performance of the garbage collectors 
+
+- Troubleshooting gc operations and policies. For example, analyzing long pauses, or determining how free memory is divided in the heap before and after a gc cycle. 
+
+When verbose garbage collection (vgc) logs are enabled, the logs are collected as soon as [garbage collection](gc.md) is initialized. 
+ 
+By default, the information is printed to STDERR. You can also [print to a file](dump_gcdump.html/#how-to-generate-a-verbonse-garbage-collection-log).
+
+You can feed verbose gc log files into variety of diagnostic tools and interfaces to visualize and analyze the garbage collection, such as the [Garbage Collection and memory visualizer extension for Eclipse](./https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv), or online services such as [GCEasy](./ https://gceasy.io). 
+
+**Note:** If vgc logs do not provide enough information to help you diagnose gc problems or perform finer tuning, you can add further information to the vgc log by calling one or more trace garbage collector (TGC) traces by using the (['-XTgc' option](xtgc.md)).
+
+## How to generate a verbose garbage collection log 
+
+You can enable the verbose gc output by using the Java&trade; virtual machine standard command line option `-verbose:gc`. For more information, see [how to use OpenJ9 virtual machine command line option](./cmdline_specifying.md).
```

Comment:
Simplify?

You can enable verbose GC logs by specifying the `-verbose:gc` standard option when you start your application. For more information, see [standard options](cmdline_general.md).

As before, you need to remove all the `./` prefixes. MkDocs doesn't require these - it can create the link by looking for that `.md` file in the current directory and all our topics are in the same `docs/` folder.

Reply:
./ prefixes removed.
Sentence simplified.